hypervisor/arch/x86/guest/vm.c

```diff
@@ -444,16 +444,16 @@ int32_t create_vm(uint16_t vm_id, struct acrn_vm_config *vm_config, struct acrn_
 			vpic_init(vm);
 		}
 
-		/* Create virtual uart;*/
-		init_vuart(vm, vm_config->vuart);
-
 		if (is_rt_vm(vm) || !is_postlaunched_vm(vm)) {
 			vrtc_init(vm);
 		}
 
-		vpci_init(vm);
+		init_vpci(vm);
```

Comment:
Should this one be after enable_iommu ?

Reply:
Yes. Will refine this.

Reply:
@dongyaozu @binbinwu1 
Seem we should enable iommu after init_vpci

commit 7e80a6eee38fdff8d1b7231414fbe3341e1b8849
Author: Binbin Wu <binbin.wu@intel.com>
Date:   Thu May 23 15:27:25 2019 +0800

    hv: vm: enable iommu after vpci init

    In current code, vpci do the pci enumartion and add pci devices to the
    context table of iommu.
    Need to enable iommu DMA address translation later than vpci init.
    Otherwise, in UEFI platform, there will be a shot time that address translation
    is enabled, but the context table is not setup.
    For the devices active in UEFI environment will have problem on address translation.